feedback-module/src/components/pages/Screen1.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import ModuleButton from "../common/Button";
+import { SCREEN1_BUTTONS } from "../../assets/constants";
+
+function Screen1({ setData, page }) {
+  const buttons = SCREEN1_BUTTONS.map(({ text, data }, index) => {
+    const handleClick = (e) => {
+      setData(data);
+      e.preventDefault();
+    };
+
+    return <ModuleButton buttonText={text} onClick={handleClick} key={index} />;
+  });
+
+  return (
+    <>
```

Comment:
Do we want to leave this as an empty div or put it in a Grid component?

Reply:
Since all screens would be wrapped in a Grid component, I decided to include the Grid part in the Modules.js part instead of here so that we wouldn't have to worry about changing the classNames in each separate file in case we ever need to